src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,30 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        while (robot.getY() != toY) {
```

Comment:
Separate this blocks to smaller methods. It's a good practice to have some small methods instead of single and giant :)